openquake/commonlib/writers.py

```diff
@@ -121,6 +121,7 @@ def write(self, descrs):
     '(slight):float32',
     '(moderate):float32',
     '(extensive):float32',
+    '(extreme):float32',
```

Comment:
The limit states defined in the fragility model need not necessarily always be ∈ {slight, moderate, extensive, extreme, complete}. They are user-defined strings.